qutebrowser/browser/mouse.py

```diff
@@ -57,7 +57,29 @@ def eventFilter(self, obj, event):
 
             if qtutils.version_check('5.11', compiled=False, exact=True):
                 # WORKAROUND for https://bugreports.qt.io/browse/QTBUG-68076
-                QTimer.singleShot(0, self._widget.setFocus)
+                try:
+                    # pylint: disable=protected-access
+                    win_id = self._widget._win_id
+                    # pylint: enable=protected-access
+                    passthrough_modes = [usertypes.KeyMode.command,
+                                         usertypes.KeyMode.prompt,
+                                         usertypes.KeyMode.yesno]
+                    if modeman.instance(win_id).mode not in passthrough_modes:
+                        tabbed_browser = objreg.get('tabbed-browser',
+                                                    scope='window',
+                                                    window=win_id)
+                        current_index = tabbed_browser.widget.currentIndex()
+                        try:
+                            widget_index = tabbed_browser.widget.indexOf(
+                                self._widget.parent())
+                        except RuntimeError:
+                            widget_index = -1
+                        if current_index == widget_index:
+                            QTimer.singleShot(0, self._widget.setFocus)
+                except:
```

Comment:
I'd rather not :wink:

![image](https://user-images.githubusercontent.com/625793/41371345-4ef22f46-6f4a-11e8-976b-01310f505da7.png)